src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1183,15 +1210,11 @@
         message))))
 
 (schema/defn ^:always-validate delete-certificate!
```

Comment:
I'd be into it. We currently don't follow the documented returns for this interface, though implementing them might be blocked on moving away from liberator for the certificate_status resource (which is SERVER-264 and in my queue).